From 2fcc2b3c3751b2fb63474ca78b24e4b6375d62a9 Mon Sep 17 00:00:00 2001
From: Richard Purdie <richard.purdie@linuxfoundation.org>
Date: Thu, 25 Feb 2021 14:52:32 +0000
Subject: [PATCH] epiphany: Fix reproducibility issue

We don't want to encide BUILD_ROOT into target packages. This is used
for build time tests but in our case those would be on target anyway
do use the target paths.

Upstream-Status: Pending
RP 2021/2/25
---
 lib/ephy-profile-utils.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/lib/ephy-profile-utils.c b/lib/ephy-profile-utils.c
index a2f6181..5218077 100644
--- a/lib/ephy-profile-utils.c
+++ b/lib/ephy-profile-utils.c
@@ -130,10 +130,10 @@ ephy_profile_utils_do_migration (const char *profile_directory,
   argv[i++] = NULL;
 
 #if DEVELOPER_MODE
-  argv[0] = BUILD_ROOT "/src/" EPHY_PROFILE_MIGRATOR;
+  argv[0] = PKGLIBEXECDIR "/" EPHY_PROFILE_MIGRATOR;
 #else
   if (debug)
-    argv[0] = BUILD_ROOT "/src/" EPHY_PROFILE_MIGRATOR;
+    argv[0] = PKGLIBEXECDIR "/" EPHY_PROFILE_MIGRATOR;
 #endif
 
   g_spawn_sync (NULL, (char **)argv, envp, G_SPAWN_SEARCH_PATH,
